include/nbl/builtin/hlsl/shapes/beziers.hlsl

```diff
@@ -412,7 +412,7 @@ struct Quadratic
         // p'(1/2) = 2(C-A)
             
         // exponent so large it would wipe the mantissa on any relative operation
-        const float_t PARAMETER_THRESHOLD = exp2(24);
+        const float_t PARAMETER_THRESHOLD = exp2(24.0f);
```

Comment:
leave a comment that it should be `exp2<float_t>(numeric_limits<float_t>::digits)` ater `tgmath` has an `exp2`